src/gleam/http.gleam

```diff
@@ -14,8 +25,10 @@ pub type Method {
   Options
   Patch
 
-  /// Non-standard but valid HTTP methods.
-  Other(String)
+  Other(
+    /// Non-standard but valid HTTP methods.
```

Comment:
We can pop this back now that the formatter bug has been fixed

Reply:
Done